dev/checkstyle.xml

```diff
@@ -52,6 +52,20 @@
       <property name="file" value="dev/checkstyle-suppressions.xml"/>
     </module>
 
+    <!--
+    If you wish to turn off checking for a section of code, you can put a comment in the source
+    before and after the section, with the following syntax:
+
+      // checkstyle:off no.XXX (such as checkstyle.off: NoFinalizer)
+      ...  // stuff that breaks the styles
+      // checkstyle:on
+    -->
+    <module name="SuppressionCommentFilter">
+        <property name="offCommentFormat" value="checkstyle.off\: ([\w\|]+)"/>
+        <property name="onCommentFormat" value="checkstyle.on\: ([\w\|]+)"/>
+        <property name="checkFormat" value="$1"/>
+    </module>
+
```

Comment:
Ah, I am sorry @ConeyLiu. I should have taken a look ahead first. I actually rather meant agreeing the previous change if we should introduce such new property... Although I (as just one of contributors) think this might be worth, maybe this is orthogonal with what the PR tries to fix.

Maybe, I think we could keep this change and then propose a new separate PR later if other people also agree with this.